.github/ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,50 @@
-> Please fill out this template when filing an issue.
-> All lines marked as quotes (`>`) instruct you with
-> what info we expect.
-> Remove this opening paragraph.
+> We LOVE to help with any issues or bug you have!
+
+> **Questions**: If you have questions about how to use Realm, please ask on 
+[SO](http://stackoverflow.com/questions/ask?tags=realm) - we monitor the Realm tag.
+
+> **Feature Request**: Just fill in the first two sections below.
+
+> **Bugs**: To help you as fast as possible with an issue or bug please describe 
+your issue and the steps you have taken to reproduce it in as many details as possible.
+>
+> Thanks for helping us help you :-)
+>
+> Please remove this and above before submitting.
 
 ## Goals
 
-> Replace this with what you tried to achieve.
+?
 
 ## Expected Results
 
-> Replace this with what you expected to happen.
+?
 
 ## Actual Results
 
-> Replace this with what actually happened.  
-> e.g. the stack trace of a crash
+?
 
 ## Steps to Reproduce
 
-> Provide steps we can follow to reproduce this issue.
+?
 
 ## Code Sample
 
-> Provide a code sample that highlights the issue.
+> Provide a code sample or test case that highlights the issue.
```

Comment:
If you can provide a test case, a PR is even better. But I don't think we can expect that. I'm confident that those who can deliver a test will be aware that filing a PR is more helpful.